### DISCUSSION TITLE
Suggestion: Change yellow light to blue, change pre signal name to pre/chain signal possibly

### DISCUSSION THREAD

**Discussion by Road-Train** [voxeltycoon/issues#750] [UI&UX]
From what I've seen in games where pre/chain signals are a thing, they will generally use blue instead of yellow as their third light. This gives off extra contrast, and makes sense coming from a real life point of view. I've seen real life signals a lot and generally a block signal will have a yellow light which means 'pass slowly'. 

Also, renaming them to either chain signals or pre/chain signals would possibly help players understand their function a bit better, because a pre-signal only implies it's placed before a signal, and doesn't quite give off the hint that you can chain pre signals together. 

Don't own the game personally but have seen some people struggle with the concept.
EDIT: if this suggestion is accepted https://github.com/voxeltycoon/issues/discussions/745 I suggest instead of changing the yellow light to blue, to instead add a blue light (giving pre signals 4 lights) and giving normal signals the pre signal look, where the yellow light is a 'reserved' light.

**Comment by victorleblais** [comment]
As a work-around, you can go in the asset editor and change the colors of any item, so you can change the color of the signal to whatever you want:
![image](https://user-images.githubusercontent.com/4365075/118121438-cc3d6d00-b3f1-11eb-8368-39f39843ecc4.png)
